contrib/examples/annotate-apply-time-mutations-simple/.expected/diff.patch

```diff
@@ -37,4 +37,4 @@ index 76ca35b..8325565 100644
      a: 0 # apply-time-mutation: ${bar.cloud.google.com/v1beta1/namespaces/example-namespace/OtherKind/example-name2:$.status.number2}
 -  stringField: placeholder # apply-time-mutation: serviceAccount:service-${resourcemanager.cnrm.cloud.google.com/namespaces/example-namespace/Project/example-name:$.status.number}@service.gserviceaccount.com
 +  stringField: serviceAccount:service-$ref2@service.gserviceaccount.com # apply-time-mutation: serviceAccount:service-${resourcemanager.cnrm.cloud.google.com/namespaces/example-namespace/Project/example-name:$.status.number}@service.gserviceaccount.com
```

Comment:
Why is this changed?

Reply:
I guess you misread it.
The only change in this file is that a newline was added at the end of the file.